src/cmd/src/cli/export.rs

```diff
@@ -42,14 +42,13 @@ type TableReference = (String, String, String);
 
 #[derive(Debug, Default, Clone, ValueEnum)]
 enum ExportTarget {
-    /// Corresponding to `SHOW CREATE TABLE`
-    #[default]
+    /// Export all table schemas, corresponding to `SHOW CREATE TABLE`.
```

Comment:
If I understand correctly, this tool is used to export data for a database. I think we can change all these three options into:
1. `schema`.
2. `data`
3. and `all`.